README.md

```diff
@@ -1,6 +1,42 @@
 # Easier68k [![Build Status](https://travis-ci.org/Chris-Johnston/Easier68k.svg?branch=master)](https://travis-ci.org/Chris-Johnston/Easier68k)
 
-A python package based assmebler, disassembler and simulator of the Motorola 68k CPU Architecture.
+Easier68k is a python library that assembles and simulates the Motorola 68k CPU Architecture.
+
+NOTE: Easier68k is currently under heavy production and is not stable.
+
+### Motivation
+
+There are very few 68K simulators and assemblers and the ones that exist were all found to be lacking in some areas.
+We realized we could do better:
+* Easier68k is cross platform.
+* Easier68k is built as a modular library so anyone can build a front end for it or integrate a 68k simulation into their project.
+* Easier68k has a focus on correctness by using unit tests and integration tests for everything.
+* Easier68k is MIT licensed so you can use it however you would like.
+
+
+### How to Use the CLI
+
+The cli is in the folder "easier68k-cli" which comes in this Github repository.
+Go to the easier68k-cli folder in the terminal/command prompt and install the dependencies with
+```bash
+python -m pip install -r requirements.txt
+```
+
+Then run the application with
+```bash
+python ./cli.py
+```
+
+You can now assemble and simulate a file
+```
+(easier68k) assemble ./file.x68, ./assembled.json
+(easier68k) simulate ./assembled.json
+(easier68k.simulate) run
```

Comment:
Not sure exactly how it's set up since I haven't taken a look at the CLI branch, but is there a CLI command in simulate to view the registers/memory/whatever? Might want to include that (unless it doesn't exist, of course)

Reply:
Yeah. the cli will have it's own readme (as the three lines down explains) with more detail. This is more to show how easy the cli works for any prospective viewers.

Reply:
Perfect.
<booming_voice>MERGE IT!</booming_voice>